### ISSUE TITLE
it doesn't work on latest ComfyUI version

### ISSUE THREAD

**Issue by donlee1** [noarche/sd-webui-color-enhance#4]
I'm getting this error with your update from 3 weeks ago:

Error occurred when executing MMakerColorEnhance:

'numpy.ndarray' object has no attribute 'mode'

File "/workspace/ComfyUI/execution.py", line 317, in execute
output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
File "/workspace/ComfyUI/execution.py", line 192, in get_output_data
return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
File "/workspace/ComfyUI/execution.py", line 169, in _map_node_over_list
process_inputs(input_dict, i)
File "/workspace/ComfyUI/execution.py", line 158, in process_inputs
results.append(getattr(obj, func)(**inputs))
File "/workspace/ComfyUI/custom_nodes/sd-webui-color-enhance/mmaker_color_enhance_comfyui.py", line 28, in apply_color_enhance
edited_image = color_enhance(edited_image.detach().cpu().numpy(), strength)
File "/workspace/ComfyUI/custom_nodes/sd-webui-color-enhance/mmaker_color_enhance_core.py", line 9, in color_enhance
if arr.mode == 'RGBA':
